### ISSUE TITLE
OSX El Capitan compilation error

### ISSUE THREAD

**Issue by daPhyre** [TiVo/intellij-haxe#370]
Using the version 0.9.7 of the plugin.

When I try to compile, it returns me the next error:

```
Information:Executing: /usr/local/lib/haxe/haxelib run lime build /Users/daphyre/haxe_test/Project.xml neko -verbose
Information:11/11/15, 15:55 - Compilation completed with 1 error and 0 warnings in 220ms
Error:Compilation failed
Information:sh: neko: command not found
```

Nevertheless, when I directly execute the line on the terminal, it works without problem, so this error might come probably from the plugin.

The SDK are pointing to these paths, and I have confirmed they exist:

Haxe toolkit home path: /usr/local/lib/haxe
Neko executable: /usr/local/lib/neko/neko
Haxelib executable: /usr/local/lib/haxe/haxelib

The problem began after updating my OSX to El Capitan, having to move haxelib from /usr/lib to /usr/local/lib because of new Apple Security Policies, which might be related to this issue.

**Comment by EBatTiVo** [comment]
@daPhyre IDEA must have the same environment as the command line for there to be an apples-to-apples (pun intended) comparison.  In this case, haxelib is responsible for running neko.  How haxelib finds the neko executable is not part of the plugin; rather it is a function of the environment.  Therefore, you must make sure that any environment variables that drive haxelib are set for idea.  One reliable way is to start idea from the same command line where you proved that haxelib works.  Starting IDEA by clicking on its desktop icon normally doesn't give the same environment.

Did you check IDEA's settings after you upgraded the OS?

**Comment by kasparsj** [comment]
a workaround is to launch IntelliJ from console:
open /Applications/IntelliJ\ IDEA\ 14.app

**Comment by tbrannam** [comment]
@kasparsj workaround seems to work with IDEA 15 as well.   I'd kind of expect the same error to occur if manually executing the same build command in the Terminal inside of IntelliJ - but that seems to work even if the build command fails.

**Comment by bmfs** [comment]
When looking for a solution for this I stumbled on this solution: http://depressiverobot.com/2016/02/05/intellij-path.html

**Comment by EBatTiVo** [comment]
That solution is very thorough and allows folks to use the GUI to launch the app.  The Haxe environment variables should also be added to the script so that all of the Haxe plumbing works as well.  If somebody wanted to make an all-inclusive version of that script, I would be happy to include it and some documentation in the github repository.

**Comment by EBatTiVo** [comment]
Should be fixed with #487.  Please reopen if version 0.10.1 of the plugin does not fix this issue.
